src/cmd/manage_nsfs.js

```diff
@@ -758,6 +780,35 @@ async function validate_account_args(data, action) {
     }
 }
 
+/**
+ * We take an object that was stringify
+ * (it unwraps ths sensitive strings, creation_date to string and removes undefined parameters)
+ * for validating against the schema we need an object, hence we parse it back to object
+ * @param {string} data
+ */
+function validate_account_schema_wrapper(data) {
```

Comment:
I suggest to avoid adding a new function only for calling a different function and throwing a different error, 
if you would like to throw ManageCLIError.InvalidSchema instead of InternalError (INVALID_SCHEMA) you can add a mapping between the two in a new RPC_ERROR_TO_MANAGE object similar to FS_ERRORS_TO_MANAGE
and in the main on manage_nsfs apply the mapping based on the err.rpc_code

Reply:
I will remove the wrapper and add the mapping as suggested, thanks.